libbeat/publisher/processing/processors.go

```diff
@@ -102,7 +102,7 @@ func (p *group) Close() error {
 }
 
 func (p *group) String() string {
-	var s []string
+	s := make([]string, len(p.list))
```

Comment:
this in conjunction with the `append` below is wrong; if you insist using `append` then you need the following `s := make([]string, 0, len(p.list))`. Also, for future reference, a more explanatory commit message other than `make lint happy` would be more helpful

Reply:
Thanks @pkoutsovasilis, I fixed it on 890e1ca7e7.
Sometimes I get sloppy when writing commit messages that only address lint issues :sweat_smile: .